Sprint-2/debug/2.js

```diff
@@ -1,14 +1,27 @@
 // Predict and explain first...
 
-const num = 103;
+// const num = 103;
 
-function getLastDigit() {
+// function getLastDigit() {
+//   return num.toString().slice(-1);
+// }
+
+// console.log(`The last digit of 42 is ${getLastDigit(42)}`);
+// console.log(`The last digit of 105 is ${getLastDigit(105)}`);
+// console.log(`The last digit of 806 is ${getLastDigit(806)}`);
+
+// This program should tell the user the last digit of each number.
+// Explain why getLastDigit is not working properly - correct the problem
+
+// correction
+// when the num is defined in the above code it uses the const it should use the let, since getLastDigit is not a constant number. 
```

Comment:
Hmm, I don't think let or const is salient here.  

Do you think you need to assign 103 to num? Why would you need to do this?